sdk/openai/openai/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+
+## 1.0.0-beta.4 (Unreleased)
+
+### Features Added
+
+- Added GetImages convenience API for image generation via `beginAzureBatchImageGeneration`, as well as supporting ImageGenerationOptions and ImageGenerationResponse aliases.
```

Comment:
merge with the other beta 3 note and change the beta3 release date to today please :)